docs/reference/rest-api/index.asciidoc

```diff
@@ -95,6 +95,7 @@ include::{xes-repo-dir}/rest-api/security.asciidoc[]
 include::{es-repo-dir}/snapshot-restore/apis/snapshot-restore-apis.asciidoc[]
 include::{es-repo-dir}/slm/apis/slm-api.asciidoc[]
 include::{es-repo-dir}/sql/apis/sql-apis.asciidoc[]
+include::{es-repo-dir}/synonyms/apis/synonyms-apis.asciidoc[]
```

Comment:
Don't forget to add it to the list too (after line 51).

Reply:
Thanks for the catch! 💯